### DISCUSSION TITLE
Trigger on low voltage

### DISCUSSION THREAD

**Discussion by gkovacsp** [fhdm-dev/scoppy#91] [Q&A]
I can't find how to trigger the capture when the voltage goes below a certain level.
My signal is a constant 3.3 V and I would like to start the capture when it goes below 0.5.

Most probably it is already possible, just can't figure it out myself.

Thanks!

**Comment by fhdm-dev** [comment]
Hi,

You can set the trigger to falling edge. You might need to the scroll to the bottom of the controls panel to see the falling edge option (on some devices it's not visible without scrolling down). 

You might also want to set the trigger mode to NORM.

There's some documentation which you might also find useful. See:
[https://oscilloscope.fhdm.xyz/app-help/Trigger-Oscilloscope](https://oscilloscope.fhdm.xyz/app-help/Trigger-Oscilloscope)

I hope that helps. Let me know it if doesn't :smile:

**Reply by gkovacsp** [reply]
I think I've figured it out but just to make sure I did: trigger is when the voltage crosses the trigger level and not when it exceeds it.

This way if the default voltage on the line is 3 V, then a trigger level of 1.5 V will start the capture when the voltage drops to 0V.

**Reply by fhdm-dev** [reply]
Yes, that's correct.

And if the trigger mode is set to AUTO then a capture will also start if no trigger event has occured after about 200ms. To prevent this you need to set the trigger mode to NORM.

**Reply by gkovacsp** [reply]
Thanks a lot!
